site/assets/scss/_anchor.scss

```diff
@@ -8,3 +8,12 @@
     text-decoration: none;
   }
 }
+
+@if $enable-prefers-reduced-motion-media-query {
```

Comment:
I would move this to `docs.scss`, `_anchor.scss` is quite a weird place to have this.

Reply:
Just did. Looks weird to me, don't you think we should add `_placeholder.scss` or something?